### ISSUE TITLE
Reliable Time not available in LUA

### ISSUE THREAD

**Issue by rainersc** [Archaegeo/DualUniverseLuaIssues#25]
In various situations we need a reliable "time" in game which is the same for all players no matter in which TZ their client runs.
Currently the time we can retrieve is depending on local real life PC time and timezone settings which makes it unusable in multiplayer use cases. as it could easily be "cheated" by the player changing local PC time or TZ settings.
Ideally just add an API to retrieve e.g. GMT time which does not take local PC time as its base.

**Comment by NQ-Ligo** [comment]
Hi there !

This issue has been fixed with 0.29 Athena update. We've added following functions and deprecated (not deleted) the getTime function for consistency :
- [float] getArkTime() : returns the current time since the arrival of the Arkship on September 30th, 2017. (equivalent to getTime)
- [float] getUtcTime(): Return the current time since January 1st, 1970.
- [float] getUtcOffset(): Return the time offset between local timezone and UTC.
